src/vhost_user/connection.rs

```diff
@@ -324,7 +325,10 @@ impl<R: Req> Endpoint<R> {
     /// * - SocketRetry: temporary error caused by signals or short of resources.
     /// * - SocketBroken: the underline socket is broken.
     /// * - SocketError: other socket related errors.
```

Comment:
What about adding the following comments like in `recv_with_fds()` documentation?

```rust
    /// # Safety
    ///
    /// It is the callers responsibility to ensure it is safe for arbitrary data to be
    /// written to the iovec pointers.
```

Reply:
Oh yes, good idea!